src/pages/TransactionInfoPage.tsx

```diff
@@ -69,18 +71,24 @@ const TransactionInfoPage = () => {
   } = useQuery({
     ...queries.transactions.transaction.one(id || ''),
     enabled: !!id,
-    refetchInterval:
-      isAppVisible && (!previousTransactionData.current || !isTxConfirmed(previousTransactionData.current))
-        ? 10000
-        : undefined
+    retry: (num, e) => {
```

Comment:
As discussed on the call:

- We need to re-add the refetchInterval (polling) logic
- We can export a numberOfAPIRetries variable from App.tsx

Reply:
done